packages/shared/tsconfig.json

```diff
@@ -14,5 +14,9 @@
         "outDir": "dist"
     },
     "include": ["src/**/*"],
-    "exclude": ["node_modules", "dist"]
+    "exclude": [
+        "dist",
+        "node_modules",
+        "src/**/*.spec.ts",
```

Comment:
We need the tests to access the tsconfig, why excluding?

Reply:
by mistake, removed it